iojs.org/aliases.json

```diff
@@ -56,28 +56,12 @@
     "myles.borins@gmail.com"
   ] },
   { "from": "report", "to": [
-    "anna@addaleax.net",
-    "info@bnoordhuis.nl",
-    "chalkerx@gmail.com",
-    "cjihrig@gmail.com",
-    "evanlucas@me.com",
-    "franziska.hinkelmann@gmail.com",
-    "fishrock123@rocketmail.com",
-    "fedor.indutny@gmail.com",
-    "jasnell@gmail.com",
-    "michael_dawson@ca.ibm.com",
-    "mscdex@mscdex.net",
-    "ofrobots@google.com",
-    "rod@vagg.org",
-    "ohtsu@ohtsu.org",
-    "trev.norris@gmail.com",
-    "rtrott@gmail.com",
-    "myles.borins@gmail.com",
-    "thechargingvolcano@gmail.com",
-    "targos@protonmail.com",
-    "joyeec9h3@gmail.com",
-    "matteo.collina@gmail.com",
-    "josh.gavant@gmail.com"
+    "tracyhinds@gmail.com",
+    "ryanharrisonlewis@gmail.com",
+    "othiym23@gmail.com",
+    "benjamingr@gmail.com",
+    "refack@gmail.com",
+    "hello@bnb.im"
```

Comment:
This can just be an alias for moderation

Reply:
There is https://github.com/nodejs/email/issues/59 (Ref: https://github.com/nodejs/email/issues/56) to consider, so maybe the duplication is Ok for now.